resources/views/components/input-field.blade.php

```diff
@@ -0,0 +1,29 @@
+@props([
+    "type" => "text",
+    "id",
+    "placeholder",
+    "variant" => "input",
+])
+
+<div class="flex flex-col gap-2 w-full max-w-[20.75rem] sm:max-w-[38rem]">
+    <label for="{{ $id }}" class="block text-base text-neutral-100 font-bold">
+        {{ $slot }}
+    </label>
+
+    @if ($variant === "textarea")
+        <textarea
+            id="{{ $id }}"
+            name="{{ $id }}"
+            placeholder="{{ $placeholder }}"
+            class="p-2 w-full min-h-16 rounded-lg bg-neutral-600 placeholder:text-neutral-200 text-base border-none focus:outline-none focus:ring-0 focus:border-transparent" 
+        ></textarea>
+    @else
+        <input 
+            type="{{ $type }}"
+            id="{{ $id }}"
+            name="{{ $id }}"
+            placeholder="{{ $placeholder }}"
+            class="p-2 w-full h-10 rounded-lg bg-neutral-600 placeholder:text-neutral-200 text-base border-none focus:outline-none focus:ring-0 focus:border-transparent"
```

Comment:
```suggestion
            class="p-2 w-full rounded-lg bg-neutral-600 placeholder:text-neutral-200 border-none focus:border-transparent"
```

Removed
- focus:ring-0 i dont think is needed
- focus:outline-none is a accessebility-feature. We can style is in css for it to look better according to our design/theme
- h-10 no need to have a locked height. It will hug by default
- text-base This is already implemented by default

This should be looked at on textarea also

Reply:
Made changes based on your commens but:

- Focus ring is needed because a blue border appears if it's not included: ![image](https://github.com/user-attachments/assets/77edf934-d259-4db6-bda2-7648f03f32a9)